pyproject.toml

```diff
@@ -46,7 +46,11 @@ looponfailroots = ["src", "tests"]
 [tool.mypy]
 python_version = "3.8"
 mypy_path = ["fixtures/stubs-for-mypy"]
-plugins = ["pydantic.mypy", "mypy_django_plugin.main", "tools.mypy_helpers.plugin"]
+plugins = [
+    "pydantic.mypy",
+    "mypy_django_plugin.main",
+    "tools.mypy_helpers.plugin",
+]
```

Comment:
unrelated change

Reply:
yeah idk why my formatter wants to do that...